src/trait-bounds.md

```diff
@@ -0,0 +1,131 @@
+# Trait and lifetime bounds
+
+> **<sup>Syntax</sup>**  
+> _TypeParamBounds_ :  
+> &nbsp;&nbsp; _TypeParamBound_ ( `+` _TypeParamBound_ )<sup>\*</sup> `+`<sup>?</sup>  
+>  
+> _TypeParamBound_ :  
+> &nbsp;&nbsp; &nbsp;&nbsp; [_Lifetime_] | _TraitBound_  
+>  
+> _TraitBound_ :  
+> &nbsp;&nbsp; &nbsp;&nbsp; `?`<sup>?</sup>[^sized_only]
```

Comment:
I don't know what the syntax you are going for with the `[^sized_only]`, but 

There's rendering issues here for what you're trying with the `[^sized_only]` but even the footnote style looks bad. Since it's only `?Sized`, why not write that into the grammar directly?

Reply:
Sized can be `use`d with another name, even if no one would ever do that. The footnotes look better in the latest version of mdbook (0.1.3). I just add the note to the prose for this though.

Reply:
I build with mdbook 0.1.3. Clicking the footnotes doesn't do anything. Perhaps because they're placed inside of the blockquote?

And right, I forgot about use aliases for `Sized`. That sort of precludes putting it in the grammar directly.